iree/samples/vision/CMakeLists.txt

```diff
@@ -44,3 +44,15 @@ iree_cc_binary(
     iree::runtime
     iree::tools::utils::image_util
 )
+
+iree_lit_test_suite(
+  NAME
+    iree_run_mnist_module_test
+  SRCS
+    "mnist_test.txt"
+  DATA
+    ::iree-run-mnist-module
+    iree::tools::IreeFileCheck
```

Comment:
Seems like the image file should be listed in data as well?

Reply:
The image file is already in the DATA field for the binary. :)